dojo/tools/eslint/parser.py

```diff
@@ -0,0 +1,59 @@
+__author__ = 'aaronweaver'
+
+import json
+from dojo.models import Finding
+
+
+class ESLintParser(object):
+    def _convert_eslint_severity_to_dojo_severity(self, eslint_severity):
+        if eslint_severity == 2:
+            return "High"
+        elif eslint_severity == 1:
+            return "Medium"
+        else:
+            return None
+
+    def __init__(self, filename, test):
+        self.items = []
+        tree = filename.read()
+        try:
+            data = json.loads(str(tree, 'utf-8'))
+        except:
+            data = json.loads(tree)
+
+        for item in data:
+            categories = ''
+            language = ''
+            mitigation = ''
+            impact = ''
+            references = ''
+            findingdetail = ''
+            title = ''
+            group = ''
+            status = ''
```

Comment:
Majority of these are unused. No point in keeping them around.

Reply:
fixed